cli/commands.go

```diff
@@ -63,11 +63,11 @@ func New() *kingpin.Application {
 		}
 	}
 
-	modName, err := GetModuleName()
+	var err error
```

Comment:
Little non-idiomatic to split it like this, no?

Reply:
Go absolutely refused to use `:=` against the preexisting `config.PackageName`, and also refused to use `=` without declaring `err` beforehand